core/embed/kernel/main.c

```diff
@@ -166,17 +169,24 @@ void drivers_init() {
 #endif
 }
 
+// defined in linker script
+extern uint32_t _codelen;
+extern uint32_t _applet_clear_ram_0_start;
```

Comment:
@TychoVrahe Not sure about this, but I think these symbols are more like `_coreapp_clear_ram_xxx` then `_aplet_clear_ram_xx`, because applet is more general.

Reply:
renamed: https://github.com/trezor/trezor-firmware/commit/6f0da0b4ccb899867f15eef0c1373be8ae38db06